### DISCUSSION TITLE
SSL For multi-level subdomain

### DISCUSSION THREAD

**Discussion by iiiusky** [projectdiscovery/interactsh#206] [Ideas]
### Interactsh version:
1.0.1

### Current Behavior:
An SSL Error Occurs When Using HTTPS to Access the Subdomain of the Applied Domain Name

### Expected Behavior:
can be accessed using https

### Steps To Reproduce:
1. Request a subdomain c7qt2vb2vtc000070630gd57bpayyyyyb.interact.sh
2. Access
https://c7qt2vb2vtc000070630gd57bpayyyyyb.interact.sh
Response can be displayed normally, but if I visit
https://aaa.c7qt2vb2vtc000070630gd57bpayyyyyb.interact.sh
Then the error message of ERR_SSL_PROTOCOL_ERROR will be prompted.

**Comment by ehsandeep** [comment]
@iiiusky this is expected behavior, as the wildcard SSL cert work for any subdomain but on same level - [more details here](https://github.com/caddyserver/caddy/issues/1578), we will also try if supporting more levels are possible as commented [here](https://github.com/caddyserver/caddy/issues/1578).

**Comment by MetzinAround** [comment]
closing old discussion
